npm/package.json

```diff
@@ -0,0 +1,38 @@
+{
+    "name": "@criticalmanufacturing/sdk",
+    "version": "1.1.2",
```

Comment:
bump version to 1.1.3

Reply:
Done

Reply:
@almeidev please also update the Version file (not only the package.json)

Reply:
Version updated (fc9dbed2785aef650f381030e870ca7c8feadd46).
Moreover, the npm package.json version is now based on the single Version.props file (3bcfbdad9294d19d4f44ebcffc038745005a382c).